### PR TITLE
feat(Sber): separate devices and add SBDV-00079

### PR DESCRIPTION
**Pull request by mrskycriper**
## TODO
- [x] add `tuyaOnOffAction` and `tuyaOnOffActionLegacy` modern extends
- [x] Move Sber devices to `sber.ts`
- [x] Add support for Smart temperature and humidity sensor (SBDV-00079)
- [x] Update images

Images https://github.com/Koenkk/zigbee2mqtt.io/pull/2679

### CODE REVIEW

File: src/lib/modernExtend.ts [hunk 1]
```diff
@@ -1464,11 +1465,17 @@ export function actionEnumLookup(args: ActionEnumLookupArgs): ModernExtend {
             if (attributeKey in msg.data) {
                 let value = getFromLookupByValue(msg.data[attributeKey], lookup);
                 // endpointNames is used when action endpoint names don't overlap with other endpoint names
-                if (args.endpointNames) value = postfixWithEndpointName(value, msg, model, meta);
+                if (args.endpointNames) {
+                    value = addEndpointName(value, msg, model, meta, args.actionTemplate === 'action_endpoint' ? 'postfix' : 'prefix');
```
Comment (Koenkk) [inline src/lib/modernExtend.ts:1469]: For consistency purposes, I would always postfix it

Comment (mrskycriper) [reply src/lib/modernExtend.ts:1469]: But what to to with existing backwards implementations then?

Comment (Koenkk) [reply src/lib/modernExtend.ts:1469]: It's only used by `tuyaOnOffAction` -> `SBDV-00032` (new device), so we don't have to consider this yet?

Comment (mrskycriper) [reply src/lib/modernExtend.ts:1469]: > It's only used by `tuyaOnOffAction` -> `SBDV-00032` (new device), so we don't have to consider this yet?

All other similar Tuya based buttons have this structure

Comment (mrskycriper) [reply src/lib/modernExtend.ts:1469]: @Koenkk tuyaOnOffAction is really just modern version of tuyaFz.on_off_action

Should I just revert all changes with postfix/prefix and use old tuyaFz.on_off_action instead?

Comment (Koenkk) [reply src/lib/modernExtend.ts:1469]: Could we have a `tuyaOnOffAction` and `tuyaLegacyOnOffAction`?

Comment (mrskycriper) [reply src/lib/modernExtend.ts:1469]: > Could we have a `tuyaOnOffAction` and `tuyaLegacyOnOffAction`?

Sounds reasonable

Comment (mrskycriper) [reply src/lib/modernExtend.ts]: Done

File: src/devices/sber.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        ],
+    },
+    {
+        fingerprint: [{modelID: 'TS0041A', manufacturerName: '_TYZB01_ub7urdza'}],
+        model: 'SBDV-00032',
+        vendor: 'Sber',
+        description: 'Smart button',
+        extend: [
+            tuyaMagicPacket(),
+            tuyaOnOffActionLegacy({actions: ['single', 'double', 'hold']}),
```
Comment (Koenkk) [inline src/devices/sber.ts:35]: Since this is a new device, why not use `tuyaOnOffAction`?

Comment (mrskycriper) [reply src/devices/sber.ts:35]: It's not new, I pulled it out of tuya whitelabels list.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**